region/norfair/east.json

```diff
@@ -5187,6 +5208,36 @@
                     }},
                     {"heatFrames": 200}
                   ]
+                },
+                {
+                  "name": "Spiky Acid Snake Frozen Platforms (Left to Right)",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canResetFallSpeed",
```

Comment:
Should there be a morphless variant of this strat, or is it necessary to be better than `Tank the Damage`?

Reply:
`canResetFallSpeed` saves a spike hit in each direction so it is necessary.